XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1315,6 +1315,30 @@
             "RelativePanel"
           ]
         },
+        {
+          "UniqueId": "Expander",
+          "Title": "Expander",
+          "Subtitle": "A container with a header that can be expanded to show a body with more content.",
+          "ImagePath": "ms-appx:///Assets/FlipView.png",
+          "Description": "The Expander has a header and can expand to show a body with more content.",
+          "Content": "Look at the <i>ExpanderPage.xaml</i> file in Visual Studio to see the full code for this page.",
+          "IsNew": true,
+          "Docs": [
+            {
+              "Title": "FlipView - API",
```

Comment:
Please update these. I suspect the actual links are not yet ready, so consider replacing these with more obvious titles/URIs like ("NYI Expander - API"/bing.com), etc.